pkg/util/file.go

```diff
@@ -48,3 +48,26 @@ func WriteFileToLog(logger *capnslog.PackageLogger, path string) {
 
 	logger.Infof("Config file %s:\n%s", path, string(contents))
 }
+
+// CreateTempFile creates a temporary file and writes
+// the given contents into the file.
+// The name of the file is returned by the function.
+// It is the callers responsibility to remove the file when no
+// longer needed.
+func CreateTempFile(contents string) (string, error) {
+	tmpfile, err := ioutil.TempFile("", "rook")
+	if err != nil {
+		return "", err
+	}
+	name := tmpfile.Name()
+	if _, err := tmpfile.WriteString(contents); err != nil {
+		_ = tmpfile.Close()
+		_ = os.Remove(name)
+		return "", err
```

Comment:
ditto

Reply:
Same here as above, since writing to the temporary file fails, that seems like the most descriptive error to return to me. Would it be an improvement to include a reference to CreateTempFile, you mean? (same question to the below ditto as well)